req1.py

```diff
@@ -1,3 +1,6 @@
-def greet(name):
-    string = 'Hello, ' + name
+def greet(name=None):
```

Comment:
the name of the file is no longer relevant, probably `greeting.py` would be better fit